### ISSUE TITLE
Better functionality for Firefox for Android

### ISSUE THREAD

**Issue by Kezxo** [pmarchwiak/auto-reader-view#3]
This seems to be the only android compatible addon that can activate the reader mode on pages where the button normally doesn't show up. However there doesn't seem to be any way to stop the addon from activating reader mode this every time apart from disabling the addon.

If this had a working enable/disable button on Firefox for android then I would be very happy.

**Comment by pmarchwiak** [comment]
Hey, thanks for the comment.

Interesting. This add-on basically just changes the URL for the current tab to the special URL used when the standard reader mode button is pressed (`about:reader?url=...`). If the normal button isn't showing up for a certain page, I'm guessing Firefox has somehow determined that page is problematic and decided not to show the button. If I understand correctly, t seems like you're just asking for a way to force reader view when Firefox doesn't show the button. 

Do you have any example URLs? What kind of behavior would you like to see instead?

**Comment by Kezxo** [comment]
> If I understand correctly, t seems like you're just asking for a way to force reader view when Firefox doesn't show the button.
Yes, and right now the only thing the addon does is forcing the reader on all pages when the addon is enabled on android and there is no way to control the addon it seems.

I just want to use reader mode on certain pages now and then, even the ones where the normal built button doesn't work.
I don't need any special stuff, just do the special url thing on any page. If the page displays or not isn't the worry.

Don't worry about adding any functionality just yet, I just want this addon to work better on android.
